src/features/pilot_management/New/pages/TemplatesPage.vue

```diff
@@ -88,6 +89,14 @@ export default Vue.extend({
     retrogradeLogo() {
       return getImagePath(ImageTag.Misc, 'retrograde_logo.png')
     },
+    selectionComplete(): boolean {
+      return this.selected !== null
+    }
+  },
+  watch: {
+    selectionComplete(bool) {
+      if (bool) window.scrollTo(0, document.body.scrollHeight)
+    },
```

Comment:
I wanted to include this `ScrollTo` watcher to bring the Template page to parity with the other Stepper pages.  That said, if anyone feels it's better to NOT scroll to bottom on template selection, please let me know along with your reasoning.

Reply:
I agree with you here. I think that we should automatically scroll to the bottom of the page on the Template page as well.